metrics-core/src/test/java/com/codahale/metrics/ConsoleReporterTest.java

```diff
@@ -216,6 +213,168 @@ public void reportsTimerValues() throws Exception {
                 ));
     }
 
+    @Test
+    public void reportMeterWithDisabledAttributes() throws Exception {
+        Set<MetricAttribute> disabledMetricAttributes = EnumSet.of(MetricAttribute.M15_RATE, MetricAttribute.M5_RATE, MetricAttribute.COUNT);
+
+        final ConsoleReporter customReporter = ConsoleReporter.forRegistry(registry)
+                .outputTo(output)
+                .formattedFor(Locale.US)
+                .withClock(clock)
+                .formattedFor(TimeZone.getTimeZone("PST"))
```

Comment:
I guess we could extract the timezone to a variable, so all tests use the same timezone.